opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/MultiFieldQuery.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.opensearch.storage.script.filter.lucene.relevance;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.opensearch.index.query.QueryBuilder;
+import org.opensearch.sql.expression.NamedArgumentExpression;
+
+/**
+ *  Base class to represent relevance queries that search multiple fields.
+ * @param <T> The builder class for the OpenSearch query.
+ */
+abstract class MultiFieldQuery<T extends QueryBuilder> extends RelevanceQuery<T> {
```

Comment:
we should have a Test file for this

Reply:
See https://github.com/Bit-Quill/opensearch-project-sql/pull/98/commits/6d79a246b94ca6699399e0bb1dc3304e16587789